app/autotranslate/server/deeplTranslate.js

```diff
@@ -151,6 +151,9 @@ class DeeplAutoTranslate extends AutoTranslate {
 						.join('\n');
 					translations[language] = this.deTokenize(Object.assign({}, message, { msg: translatedText }));
 				}
+				if (result.statusCode === 456) {
```

Comment:
Hey! The docs says that `HTTP` client should throw an error if the status code is 4xx or 5xx. Do you know if it's happening here? 
In that case, the `catch` should handle the logic in this `if`.